test/e2e/web-map/mapAttributes.test.js

```diff
@@ -0,0 +1,37 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("Adding Width and Height Attribute to mapml-viewer", () => {
+  let page;
+  let context;
+  test.beforeAll(async function() {
+    context = await chromium.launchPersistentContext('');
+    page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+    await page.goto("mapAttributes.html");
+  });
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test("Setting Opacity Attibute to mapml-viewer", async () => {
+    let height_attribute_value = await page.$eval("body > map",(val) => val.getAttribute("height"));
+    let width_attribute_value = await page.$eval("body > map",(val) => val.getAttribute("width"));
+    let height;
+    let width;
+    if (!height_attribute_value && !width_attribute_value){
+        return;
```

Comment:
replace this return with expects for the value of height_attribute_value and width_attribute_value.  If they fail, there was something wrong with the initial conditions, and you'd want to know about that.